docs/fpp-users-guide.html

```diff
@@ -6089,7 +6086,7 @@ <h4 id="Defining-Components_Port-Instances_Serial-Port-Instances">10.2.6. Serial
 This flexibility comes at the cost that you lose the type
 compile-time type checking provided by port connections with named types.
 For more information about serial ports and their use, see
-the <a href="https://fprime.jpl.nasa.gov/latest/documentation/user-manual/">F Prime User Manual</a>.</p>
+the <a href="https://fprime.jpl.nasa.gov/devel/documentation/user-manual/">F Prime User Manual</a>.</p>
```

Comment:
One change that is coming out soon is `documentation` will be replaced by `docs` in the URLs, as it will be mirroring the repo's structure. I am hoping to have this live this week. Up to you if you want to hold or not.

Reply:
Fixed! Thanks.